client/my-sites/stats/stats-purchase/stats-purchase-tier-upgrade-slider.tsx

```diff
@@ -54,7 +54,7 @@ function TierUpgradeSlider( { className, currencyCode }: TierUpgradeSliderProps
 	// TODO: Review tier values from API.
 	// Should consider validating the inputs before displaying them.
 	// The following will draw a "-" for the views value if it's undefined.
-	const hasExtension = tiers[ currentPlanIndex ]?.extension || false;
+	const hasExtension = !! tiers[ currentPlanIndex ]?.extension;
```

Comment:
👍🏼

Reply:
@a8ck3n this is a ideological pattern where we want boolean 😄